controllers/datadogagent/clusteragent.go

```diff
@@ -1107,6 +1106,16 @@ func getDefaultClusterAgentPolicyRules() []rbacv1.PolicyRule {
 				datadoghqv1alpha1.WatchVerb,
 			},
 		},
+		{
+			APIGroups: []string{datadoghqv1alpha1.CoreAPIGroup},
+			Resources: []string{datadoghqv1alpha1.EventsResource},
+			Verbs: []string{
+				datadoghqv1alpha1.GetVerb,
+				datadoghqv1alpha1.ListVerb,
+				datadoghqv1alpha1.WatchVerb,
+				datadoghqv1alpha1.CreateVerb, // Needed when the external metrics opt is enabled
+			},
+		},
```

Comment:
IMO you can keep event in the previous rule.
but create a new rules if the `external metrics opt` is enabled.

It is not a problem to have 2 rules for the same resources. the final result will be a merge of the 2 .

Reply:
Changed :+1: